clover.rb

```diff
@@ -573,11 +573,7 @@ def assets
   hash_branch("after-login") do |r|
     r.get web? do
       no_authorization_needed
-      if (project = current_account.projects_dataset.order(:created_at, :name).first)
-        r.redirect "#{project.path}/dashboard"
-      else
-        r.redirect "/project"
-      end
+      default_project_dashboard_redirect
```

Comment:
It sounds like returning a value.
```suggestion
      redirect_default_project_dashboard
```
OR
```suggestion
      request.redirect default_project_dashboard
```

Reply:
Pushed a commit to switch to `redirect_default_project_dashboard`.